caesars-palace/blocks/form/form.css

```diff
@@ -33,29 +33,29 @@ main .form select {
   margin-top: 4px;
 }
 
-main .form textarea {
+.form textarea {
   min-height: 100px;
 }
 
-main .form.sign-in input,
-main .form.sign-in textarea,
-main .form.sign-in select {
+.form.sign-in input,
+.form.sign-in textarea,
+.form.sign-in select {
   padding: 16px 16px 4px;
-  font: inherit;
+  font: 14px;
 }
 
-main .form.sign-in input.error,
-main .form.sign-in textarea.error,
-main .form.sign-in select.error { 
+.form.sign-in input.error,
+.form.sign-in textarea.error,
+.form.sign-in select.error { 
   border-color: #d60000;
```

Comment:
Can you try to keep the notation of colors consistent? All rgb?

Reply:
Done